remoting/rico-remoting-common/src/test/java/dev/rico/internal/remoting/converters/ZonedDateTimeConverterTest.java

```diff
@@ -1,127 +1,152 @@
-        final ZonedDateTime time = ZonedDateTime.now();
-        final Converter converter = factory.getConverterForType(ZonedDateTime.class);
-
-        //when
-        final Object rawObject = converter.convertToRemoting(time);
-        final Object reConverted = converter.convertFromRemoting(rawObject);
-
-        //then
-        Assert.assertNotNull(rawObject);
-        Assert.assertNotNull(reConverted);
-        Assert.assertTrue(ZonedDateTime.class.isAssignableFrom(reConverted.getClass()));
-        final ZonedDateTime reconvertedTime = (ZonedDateTime) reConverted;
-        Assert.assertEquals(reconvertedTime, time);
-    }
-
-    @Test
-    public void testDifferentTimeZone() throws ValueConverterException {
-        //given
-        final ZonedDateTimeConverterFactory factory = new ZonedDateTimeConverterFactory();
-        final ZoneId currentZoneId = ZoneId.systemDefault();
-        final ZoneId differentZoneId = ZoneId.getAvailableZoneIds().stream()
-                .map(i -> ZoneId.of(i))
-                .filter(zoneId -> !Objects.equals(zoneId, currentZoneId))
-                .findAny()
-                .orElseThrow(() -> new RuntimeException("No Zone ID found!"));
-        final ZonedDateTime time = ZonedDateTime.now(differentZoneId);
-        final Converter converter = factory.getConverterForType(ZonedDateTime.class);
-
-        //when
-        final Object rawObject = converter.convertToRemoting(time);
-        System.out.println(rawObject);
-        final Object reConverted = converter.convertFromRemoting(rawObject);
-
-        //then
-        Assert.assertNotNull(rawObject);
-        Assert.assertNotNull(reConverted);
-        Assert.assertTrue(ZonedDateTime.class.isAssignableFrom(reConverted.getClass()));
-        final ZonedDateTime reconvertedTime = (ZonedDateTime) reConverted;
-        Assert.assertEquals(reconvertedTime, time);
-    }
-
-    @Test(enabled = false)
-    public void testRawSameTimeZone() throws ValueConverterException, ParseException {
-        final TimeZone defaultZone = TimeZone.getDefault();
-        try {
-
-            //given
-            final ZonedDateTimeConverterFactory zonedDateTimeFactory = new ZonedDateTimeConverterFactory();
-            final Converter zonedDateTimeConverter = zonedDateTimeFactory.getConverterForType(LocalDate.class);
-            final String rawObject = "2019-01-30T11:25:07.341+08:00";
-
-            //when
-            final Object reconverted = zonedDateTimeConverter.convertFromRemoting(rawObject);
-
-            //then
-            Assert.assertNotNull(reconverted);
-            Assert.assertTrue(ZonedDateTime.class.isAssignableFrom(reconverted.getClass()));
-            final ZonedDateTime reconvertedTime = (ZonedDateTime) reconverted;
-            Assert.assertEquals(ZoneId.of("Australia/Sydney"), reconvertedTime.getZone());
-
-        } finally {
-            TimeZone.setDefault(defaultZone);
-        }
-    }
-
-}
+/*

```

Comment:
what did change here ??

Reply:
I think the change to the java file should be moved to a separate PR

Reply:
will do